lib/sedna/core/federated_learning/federated_learning.py

```diff
@@ -13,14 +13,20 @@
 # limitations under the License.
 
 
+import asyncio
+import sys
 import time
 
-from sedna.core.base import JobBase
-from sedna.common.config import Context
-from sedna.common.file_ops import FileOps
+from plato.clients import registry as client_registry
+from plato.config import Config
```

Comment:
As plato is not used as a basic dependency, it is recommended to move it to `__init__` of `FederatedLearningV2`.

Reply:
add ```plato-learn``` as a basic dependency in requirements.txt.